### PR TITLE
[Documentation] Documentation for API routings

### PR DESCRIPTION
**Pull request by tuka217**
| Q               | A
| --------------- | ---
| Bug fix?        | no|
| New feature?    | no|
| BC breaks?      | no|
| Related tickets | based on #7482||
| License         | MIT |

### CODE REVIEW

File: docs/api/product_variants.rst [hunk 1]
```diff
@@ -692,35 +648,35 @@ Example Response
...
 +-------------------------------------+----------------+----------------------------------------------------------+
 | translations['locale_code']['name'] | request        | Name of product variant                                  |
 +-------------------------------------+----------------+----------------------------------------------------------+
 
 Example
-.......
+^^^^^^^
 
-To partial update the product variant with id equals to 342 for the product with id equals to 1 use the below method.
+To partial update the product variant with ``code = double-theme-mug`` for the product with ``code = MUG-TH`` use the below method.
```
Comment (TheMadeleine) [inline docs/api/product_variants.rst]: To partially update

File: docs/api/product_variants.rst [hunk 2]
```diff
@@ -744,33 +700,33 @@ Example Response
 Deleting Product Variant
 ------------------------
 
-To delete a product variant you will need to call the ``/api/v1/products/product_id/variants/variant_id`` endpoint with ``DELETE`` method.
+To delete a product variant you will need to call the ``/api/v1/products/productCode/variants/code`` endpoint with ``DELETE`` method.
```
Comment (TheMadeleine) [inline docs/api/product_variants.rst]: ``` with the ``DELETE`` method.```

File: docs/api/product_variants.rst [hunk 3]
```diff
@@ -744,33 +700,33 @@ Example Response
...
-| productId                           | url attribute  | Id of product for which the variants should be displayed |
+| productCode                         | url attribute  | Id of product for which the variants should be displayed |
 +-------------------------------------+----------------+----------------------------------------------------------+
 
 Example
-.......
+^^^^^^^
 
-To delete the product variant with id equals to 333 from product with id equals to 1 use the method below.
+To delete the product variant with ``code = double-theme-mug`` from product with ``code = MUG-TH`` use the method below.
```
Comment (TheMadeleine) [inline docs/api/product_variants.rst]: from the product with

Comment (TheMadeleine) [inline docs/api/product_variants.rst]: use the below method.

File: docs/api/products.rst [hunk 4]
```diff
@@ -569,36 +644,37 @@ Exemplary Response
 Getting a Single Product
 ------------------------
 
-To retrieve the details of the product you will need to call the ``/api/v1/product/id`` endpoint with the ``GET`` method.
+To retrieve the details of the product you will need to call the ``/api/v1/product/code`` endpoint with the ``GET`` method.
```
Comment (TheMadeleine) [inline docs/api/products.rst]: of a product

File: docs/api/products.rst [hunk 5]
```diff
@@ -569,36 +644,37 @@ Exemplary Response
...
 +===============+================+======================================+
 | Authorization | header         | Token received during authentication |
 +---------------+----------------+--------------------------------------+
-| id            | url attribute  | Id of requested resource             |
+| code          | url attribute  | Unique product identifier            |
 +---------------+----------------+--------------------------------------+
 
 Example
 ^^^^^^^
+To see the details for the the product with ``code = spoon`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/products.rst]: details of the product

File: docs/api/products.rst [hunk 6]
```diff
@@ -569,36 +644,37 @@ Exemplary Response
...
-	$ curl http://demo.sylius.org/api/v1/products/2 \
+	$ curl http://demo.sylius.org/api/v1/products/spoon \
 		-H "Authorization: Bearer SampleToken" \
 		-H "Accept: application/json"
 
 .. note::
 
-	*2* is an exemplary value. Your value can be different.
-	Check in the list of all products if you are not sure which id should be used.
+	*spoon* is an exemplary value. Your value can be different.
```
Comment (TheMadeleine) [inline docs/api/products.rst]: `The *spoon* code`

File: docs/api/promotion_coupons.rst [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+.. code-block:: text
+
+    GET /api/v1/promotions/{promotionCode}/coupons/{code}
+
++---------------+----------------+---------------------------------------------------+
+| Parameter     | Parameter type | Description                                       |
++===============+================+===================================================+
+| Authorization | header         | Token received during authentication              |
++---------------+----------------+---------------------------------------------------+
+| code          | url attribute  | Code of requested promotion                       |
```
Comment (TheMadeleine) [inline docs/api/promotion_coupons.rst]: the requested <del>promotion</del> **coupon** :tada:

File: docs/api/promotion_coupons.rst [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
++---------------+----------------+---------------------------------------------------+
+| promotionCode | url attribute  | Code of promotion to which the coupon is assigned |
++---------------+----------------+---------------------------------------------------+
+
+Example
+^^^^^^^
+
+To see the details for the the promotion coupon with ``code = A3BCB`` which belongs to promotion with ``code = autumnal_promo`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/promotion_coupons.rst]: of the promotion coupon

File: docs/api/promotion_coupons.rst [hunk 9]
```diff
@@ -0,0 +1,229 @@
...
+
+.. code-block:: bash
+
+     $ curl http://demo.sylius.org/api/v1/promotions/autumnal_promo/coupons/A3BCB \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json"
+
+.. note::
+
+    *A3BCB* and *autumnal_promo* are just examples. Your value can be different.
```
Comment (TheMadeleine) [inline docs/api/promotion_coupons.rst]: `The *A3BCB* and *autumnal_promo* codes`

File: docs/api/promotion_coupons.rst [hunk 10]
```diff
@@ -0,0 +1,229 @@
...
+| Authorization | header         | Token received during authentication                              |
++---------------+----------------+-------------------------------------------------------------------+
+| promotionCode | url attribute  | Code of promotion to which the coupons are assigned               |
++---------------+----------------+-------------------------------------------------------------------+
+| page          | query          | *(optional)* Number of the page, by default = 1                   |
++---------------+----------------+-------------------------------------------------------------------+
+| paginate      | query          | *(optional)* Number of items to display per page, by default = 10 |
++---------------+----------------+-------------------------------------------------------------------+
+
+To see the first page of all promotion coupons assigned to promotion with ``code = autumnal_promo`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/promotion_coupons.rst]: assigned to the promotion

File: docs/api/promotions.rst [hunk 11]
```diff
@@ -1,186 +1,201 @@
...
++-------------+-------------------------------------------------------------+
+| usageLimit  | Promotion's usage limit                                     |
++-------------+-------------------------------------------------------------+
+| used        | Number of times this promotion has been used                |
++-------------+-------------------------------------------------------------+
+| priority    | When exclusive, promotion with top priority will be applied |
++-------------+-------------------------------------------------------------+
+| couponBased | Whether this promotion is triggered by a coupon             |
++-------------+-------------------------------------------------------------+
+| exclusive   | Cannot be applied together with other promotions            |
```
Comment (TheMadeleine) [inline docs/api/promotions.rst]: When true the promotion cannot be applied together with other promotions

File: docs/api/promotions.rst [hunk 12]
```diff
@@ -1,186 +1,201 @@
...
+| createdAt   | Date of creation                                            |
++-------------+-------------------------------------------------------------+
+| updatedAt   | Date of last update                                         |
++-------------+-------------------------------------------------------------+
+| channels    | Collection of channels in which the promotion is available  |
++-------------+-------------------------------------------------------------+
+
+.. note::
+
+    Read more about :doc:`Promotion in the component docs</components/Promotion/models>`.
```
Comment (TheMadeleine) [inline docs/api/promotions.rst]: Promotions

File: docs/api/promotions.rst [hunk 13]
```diff
@@ -1,186 +1,201 @@
...
+
+Getting a Single Promotion
+--------------------------
 
-page
-    Number of the page, by default = 1
-limit
-    Number of items to display per page
+To retrieve the details of the promotion you will need to call the ``/api/v1/promotions/code`` endpoint with the ``GET`` method.
```
Comment (TheMadeleine) [inline docs/api/promotions.rst]: details of a promotion

File: docs/api/promotions.rst [hunk 14]
```diff
@@ -1,186 +1,201 @@
...
-    STATUS: 200 OK
+    GET /api/v1/promotions/{code}
 
-.. code-block:: json
++---------------+----------------+--------------------------------------+
+| Parameter     | Parameter type | Description                          |
++===============+================+======================================+
+| Authorization | header         | Token received during authentication |
++---------------+----------------+--------------------------------------+
+| code          | url attribute  | Code of requested promotion          |
```
Comment (TheMadeleine) [inline docs/api/promotions.rst]: the requested promotion

File: docs/api/promotions.rst [hunk 15]
```diff
@@ -1,186 +1,201 @@
...
-                }
-            ]
-        }
-    }
+Example
+^^^^^^^
 
-Getting a single promotion
---------------------------
+To see the details for the the promotion with ``code = christmas`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/promotions.rst]: details of the promotion

File: docs/api/promotions.rst [hunk 16]
```diff
@@ -1,186 +1,201 @@
...
 
-.. code-block:: text
+     $ curl http://demo.sylius.org/api/v1/promotions/christmas \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json"
+
+.. note::
 
-    GET /api/v1/promotions/1
+    *christmas* is just an example. Your value can be different.
```
Comment (TheMadeleine) [inline docs/api/promotions.rst]: `The *christmas* code`

File: docs/api/promotions.rst [hunk 17]
```diff
@@ -1,186 +1,201 @@
...
 
-Response
-~~~~~~~~
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
 
 .. code-block:: text
 
-    STATUS: 200 OK
+     STATUS: 200 OK
```
Comment (TheMadeleine) [inline docs/api/promotions.rst]: is this additional space needed here?

File: docs/api/shipping_methods.rst [hunk 18]
```diff
@@ -0,0 +1,105 @@
...
++---------------------+----------------------------------------------------+
+| Field               | Description                                        |
++=====================+====================================================+
+| id                  | Id of the shipping method                          |
++---------------------+----------------------------------------------------+
+| code                | Unique shipping method identifier                  |
++---------------------+----------------------------------------------------+
+| name                | The name of the shipping method                    |
++---------------------+----------------------------------------------------+
+| enabled             | Determine if shipping method is enabled            |
```
Comment (TheMadeleine) [inline docs/api/shipping_methods.rst]: the shipping method

File: docs/api/shipping_methods.rst [hunk 19]
```diff
@@ -0,0 +1,105 @@
...
++---------------------+----------------------------------------------------+
+| categoryRequirement | Reference to constant from ShippingMethodInterface |
++---------------------+----------------------------------------------------+
+| calculator          | Reference to constant from DefaultCalculators      |
++---------------------+----------------------------------------------------+
+| configuration       | Extra configuration for calculator                 |
```
Comment (TheMadeleine) [inline docs/api/shipping_methods.rst]: for the calculator

File: docs/api/shipping_methods.rst [hunk 20]
```diff
@@ -0,0 +1,105 @@
...
++---------------------+----------------------------------------------------+
+| createdAt           | Date of creation                                   |
++---------------------+----------------------------------------------------+
+| updatedAt           | Date of last update                                |
++---------------------+----------------------------------------------------+
+
+.. note::
+
+    Read more about :doc:`Shipping Method in the component docs</components/Shipping/models>`.
```
Comment (TheMadeleine) [inline docs/api/shipping_methods.rst]: Shipping Methods

File: docs/api/shipping_methods.rst [hunk 21]
```diff
@@ -0,0 +1,105 @@
...
+
+Getting a Single Shipping Method
+--------------------------------
+
+To retrieve the details of the shipping method you will need to call the ``/api/v1/shipping-methods/code`` endpoint with the ``GET`` method.
```
Comment (TheMadeleine) [inline docs/api/shipping_methods.rst]: details of a shipping method

File: docs/api/shipping_methods.rst [hunk 22]
```diff
@@ -0,0 +1,105 @@
...
+.. code-block:: text
+
+    GET /api/v1/shipping-methods/{code}
+
++---------------+----------------+--------------------------------------+
+| Parameter     | Parameter type | Description                          |
++===============+================+======================================+
+| Authorization | header         | Token received during authentication |
++---------------+----------------+--------------------------------------+
+| code          | url attribute  | Code of requested shipping method    |
```
Comment (TheMadeleine) [inline docs/api/shipping_methods.rst]: the requested shipping method

File: docs/api/shipping_methods.rst [hunk 23]
```diff
@@ -0,0 +1,105 @@
...
++---------------+----------------+--------------------------------------+
+
+Example
+^^^^^^^
+
+To see the details for the the shipping method with ``code = ups`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/shipping_methods.rst]: details of the shipping method

File: docs/api/shipping_methods.rst [hunk 24]
```diff
@@ -0,0 +1,105 @@
...
+
+.. code-block:: bash
+
+     $ curl http://demo.sylius.org/api/v1/shipping-methods/ups \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json"
+
+.. note::
+
+    *ups* is just an example. Your value can be different.
```
Comment (TheMadeleine) [inline docs/api/shipping_methods.rst]: `The *ups* code`

File: docs/api/tax_categories.rst [hunk 25]
```diff
@@ -0,0 +1,532 @@
...
+| description | Description of the tax category               |
++-------------+-----------------------------------------------+
+| createdAt   | Date of creation                              |
++-------------+-----------------------------------------------+
+| updatedAt   | Date of last update                           |
++-------------+-----------------------------------------------+
+
+.. note::
+
+	Read more about :doc: `Tax Category model in the component docs</components/Taxation/models>`.
```
Comment (TheMadeleine) [inline docs/api/tax_categories.rst]: the Tax Category model

Comment (TheMadeleine) [inline docs/api/tax_categories.rst]: and the link will not work should be-> ```:doc:` ... ` ```

File: docs/api/tax_categories.rst [hunk 26]
```diff
@@ -0,0 +1,532 @@
...
+			"self": {
+				"href": "\/api\/v1\/tax-categories\/food"
+			}
+		}
+	}
+
+Getting a Single Tax Category
+-----------------------------
+
+To retrieve the details of the tax category you will need to call the ``/api/v1/tax-categories/code`` endpoint with the ``GET`` method.
```
Comment (TheMadeleine) [inline docs/api/tax_categories.rst]: details of a tax category

File: docs/api/tax_categories.rst [hunk 27]
```diff
@@ -0,0 +1,532 @@
...
+
+.. code-block:: bash
+
+	$ curl http://demo.sylius.org/api/v1/tax-categories/food \
+		-H "Authorization: Bearer SampleToken" \
+		-H "Accept: application/json"
+
+.. note::
+
+	*food* is an exemplary value. Your value can be different.
```
Comment (TheMadeleine) [inline docs/api/tax_categories.rst]: `The *food** code`

File: docs/api/tax_categories.rst [hunk 28]
```diff
@@ -0,0 +1,532 @@
...
+					}
+				}
+			]
+		}
+	}
+
+Updating a Tax Category
+-----------------------
+
+To fully update a tax category you will need to call the ``/api/v1/tax-categories/code`` endpoint with ``PUT`` method.
```
Comment (TheMadeleine) [inline docs/api/tax_categories.rst]: ```with the ``PUT`` method```

File: docs/api/tax_categories.rst [hunk 29]
```diff
@@ -0,0 +1,532 @@
...
+		'
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+	STATUS: 204 No Content
+
+If you try to perform a full tax category update without all required fields specified, you will receive a ``400 Bad Request`` error.
```
Comment (TheMadeleine) [inline docs/api/tax_categories.rst]: all **the** required fields specified

File: docs/api/tax_rates.rst [hunk 30]
```diff
@@ -0,0 +1,229 @@
...
+| calculator      | Type of calculator                 |
++-----------------+------------------------------------+
+| createdAt       | Date of creation                   |
++-----------------+------------------------------------+
+| updatedAt       | Date of last update                |
++-----------------+------------------------------------+
+
+.. note::
+
+    Read more about :doc:`Tax Rate in the component docs</components/Taxation/models>`.
```
Comment (TheMadeleine) [inline docs/api/tax_rates.rst]: Tax Rates

File: docs/api/tax_rates.rst [hunk 31]
```diff
@@ -0,0 +1,229 @@
...
+
+Getting a Single Tax Rate
+--------------------------
+
+To retrieve the details of the tax rate you will need to call the ``/api/v1/tax-rates/code`` endpoint with the ``GET`` method.
```
Comment (TheMadeleine) [inline docs/api/tax_rates.rst]: details of a tax rate

File: docs/api/tax_rates.rst [hunk 32]
```diff
@@ -0,0 +1,229 @@
...
+.. code-block:: text
+
+    GET /api/v1/tax-rates/{code}
+
++---------------+----------------+--------------------------------------+
+| Parameter     | Parameter type | Description                          |
++===============+================+======================================+
+| Authorization | header         | Token received during authentication |
++---------------+----------------+--------------------------------------+
+| code          | url attribute  | Code of requested tax rate           |
```
Comment (TheMadeleine) [inline docs/api/tax_rates.rst]: the requested tax rate

File: docs/api/tax_rates.rst [hunk 33]
```diff
@@ -0,0 +1,229 @@
...
++---------------+----------------+--------------------------------------+
+
+Example
+^^^^^^^
+
+To see the details for the the tax rate with ``code = clothing_sales_tax_7`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/tax_rates.rst]: details of the tax rate

File: docs/api/tax_rates.rst [hunk 34]
```diff
@@ -0,0 +1,229 @@
...
+
+.. code-block:: bash
+
+     $ curl http://demo.sylius.org/api/v1/tax-rates/clothing_sales_tax_7 \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json"
+
+.. note::
+
+    *clothing_sales_tax_7* is just an example. Your value can be different.
```
Comment (TheMadeleine) [inline docs/api/tax_rates.rst]: `The *clothing_sales_tax_7* code`

File: docs/api/taxons.rst [hunk 35]
```diff
@@ -351,37 +351,37 @@ Exemplary Response
 Getting a Single Taxon
 ----------------------
 
-To retrieve the details of the taxon you will need to call the ``/api/v1/taxons/taxon_id`` endpoint with the ``GET`` method.
+To retrieve the details of the taxon you will need to call the ``/api/v1/taxons/code`` endpoint with the ``GET`` method.
```
Comment (TheMadeleine) [inline docs/api/taxons.rst]: details of a taxon

File: docs/api/taxons.rst [hunk 36]
```diff
@@ -351,37 +351,37 @@ Exemplary Response
...
 +---------------+----------------+--------------------------------------+
-| id            | url attribute  | Id of requested taxon                |
+| code          | url attribute  | Identifier of the requested taxon    |
 +---------------+----------------+--------------------------------------+
 
 Example
 ^^^^^^^
 
-To see the details for the the taxon with ``id = 9`` use the below method:
+To see the details for the the taxon with ``code = toys`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/taxons.rst]: details of the taxon

File: docs/api/taxons.rst [hunk 37]
```diff
@@ -787,41 +789,41 @@ Exemplary Response
 Updating Taxon
 --------------
 
-To fully update a taxon you will need to call the ``/api/v1/taxons/taxon_id`` endpoint with ``PUT`` method.
+To fully update a taxon you will need to call the ``/api/v1/taxons/code`` endpoint with ``PUT`` method.
```
Comment (TheMadeleine) [inline docs/api/taxons.rst]: ``` with the ``PUT`` method.```

File: docs/api/taxons.rst [hunk 38]
```diff
@@ -787,41 +789,41 @@ Exemplary Response
...
-| images                                     | request        | *(optional)* Images codes assigned to the taxon |
-+--------------------------------------------+----------------+-------------------------------------------------+
+    PUT /api/v1/taxons/{code}
+
++--------------------------------------------+----------------+----------------------------------------------------+
+| Parameter                                  | Parameter type | Description                                        |
++============================================+================+====================================================+
+| Authorization                              | header         | Token received during authentication               |
++--------------------------------------------+----------------+----------------------------------------------------+
+| code                                       | url attribute  | **(unique)** Identifier of the requested the taxon |
```
Comment (TheMadeleine) [inline docs/api/taxons.rst]: of the requested <del>the</del> taxon

File: docs/api/taxons.rst [hunk 39]
```diff
@@ -787,41 +789,41 @@ Exemplary Response
...
+| parent                                     | request        | *(optional)* The parent taxon's code               |
++--------------------------------------------+----------------+----------------------------------------------------+
+| images                                     | request        | *(optional)* Images codes assigned to the taxon    |
++--------------------------------------------+----------------+----------------------------------------------------+
 
 Example
 ^^^^^^^
 
-To full update the taxon with ``id = 9`` use the below method:
+To full update the taxon with ``code = toys`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/taxons.rst]: To fully update

File: docs/api/taxons.rst [hunk 40]
```diff
@@ -843,31 +845,31 @@ Exemplary Response
...
-| Authorization | header         | Token received during authentication |
-+---------------+----------------+--------------------------------------+
-| id            | url attribute  | Id of the requested taxon            |
-+---------------+----------------+--------------------------------------+
++---------------+----------------+----------------------------------------------------+
+| Parameter     | Parameter type | Description                                        |
++===============+================+====================================================+
+| Authorization | header         | Token received during authentication               |
++---------------+----------------+----------------------------------------------------+
+| code          | url attribute  | **(unique)** Identifier of the requested the taxon |
```
Comment (TheMadeleine) [inline docs/api/taxons.rst]: <del>the</del> taxon

File: docs/api/taxons.rst [hunk 41]
```diff
@@ -843,31 +845,31 @@ Exemplary Response
...
++---------------+----------------+----------------------------------------------------+
 
 Example
 ^^^^^^^
 
-To partial update the taxon with ``id = 9`` use the below method:
+To partial update the taxon with ``code = toys`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/taxons.rst]: To partially update

File: docs/api/taxons.rst [hunk 42]
```diff
@@ -901,29 +903,29 @@ Definition
...
-| Authorization | header         | Token received during authentication |
-+---------------+----------------+--------------------------------------+
-| id            | url attribute  | Id of the taxon to be removed        |
-+---------------+----------------+--------------------------------------+
++---------------+----------------+----------------------------------------------------+
+| Parameter     | Parameter type | Description                                        |
++===============+================+====================================================+
+| Authorization | header         | Token received during authentication               |
++---------------+----------------+----------------------------------------------------+
+| code          | url attribute  | **(unique)** Identifier of the requested the taxon |
```
Comment (TheMadeleine) [inline docs/api/taxons.rst]: <del>the</del> taxon

File: docs/api/zones.rst [hunk 43]
```diff
@@ -0,0 +1,491 @@
...
+| type      | Type of the zone       |
++-----------+------------------------+
+| createdAt | Date of creation       |
++-----------+------------------------+
+| updatedAt | Date of last update    |
++-----------+------------------------+
+
+.. note::
+
+	Read more about :doc: `Zone model in the component docs</components/Addressing/models>`.
```
Comment (TheMadeleine) [inline docs/api/zones.rst]: ```:doc:`...` ```
the Zone model

File: docs/api/zones.rst [hunk 44]
```diff
@@ -0,0 +1,491 @@
...
+				},
+				"members": {}
+			}
+		}
+	}
+
+Getting a Single Zone
+---------------------
+
+To retrieve the details of the zone you will need to call the ``/api/v1/zone/code`` endpoint with the ``GET`` method.
```
Comment (TheMadeleine) [inline docs/api/zones.rst]: details of a zone

File: docs/api/zones.rst [hunk 45]
```diff
@@ -0,0 +1,491 @@
...
++===============+================+======================================+
+| Authorization | header         | Token received during authentication |
++---------------+----------------+--------------------------------------+
+| code          | url attribute  | Unique zone identifier               |
++---------------+----------------+--------------------------------------+
+
+Example
+^^^^^^^
+
+To see the details for the the zone with ``code = EU`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/zones.rst]: details of the zone

File: docs/api/zones.rst [hunk 46]
```diff
@@ -0,0 +1,491 @@
...
+
+.. code-block:: bash
+
+	$ curl http://demo.sylius.org/api/v1/zones/EU \
+		-H "Authorization: Bearer SampleToken" \
+		-H "Accept: application/json"
+
+.. note::
+
+	*EU* is an exemplary value. Your value can be different.
```
Comment (TheMadeleine) [inline docs/api/zones.rst]: `The *EU* code`

File: docs/api/zones.rst [hunk 47]
```diff
@@ -0,0 +1,491 @@
...
+					}
+				}
+			]
+		}
+	}
+
+Updating a Zone
+---------------
+
+To fully update a zone you will need to call the ``/api/v1/zones/code`` endpoint with ``PUT`` method.
```
Comment (TheMadeleine) [inline docs/api/zones.rst]: ``` with the ``PUT`` method.````

File: docs/api/zones.rst [hunk 48]
```diff
@@ -0,0 +1,491 @@
...
+					"errors": [
+						"Please enter the scope."
+					]
+				},
+				"code": {},
+				"members": {}
+			}
+		}
+	}
+
```
Comment (TheMadeleine) [inline docs/api/zones.rst]: redundant line

File: docs/api/product_variants.rst [hunk 49]
```diff
@@ -744,33 +700,33 @@ Example Response
 Deleting Product Variant
```
Comment (TheMadeleine) [inline docs/api/product_variants.rst]: Deleting **a** Product Variant

File: docs/api/zones.rst (hunk 48 above)
Comment (tuka217) [inline docs/api/zones.rst]: Done

File: docs/api/product_variants.rst [hunk 50]
```diff
@@ -1,358 +1,113 @@
...
-        "_links": {
-            "self": {
-                "href": "/api/v1/products/1/variants/1"
-            },
-            "product": {
-                "href": "/api/v1/products/1"
-            }
-        }
-    }
+    Read more about :doc:`ProductVariant model in the component docs</components/Product/models>`.
```
Comment (TheMadeleine) [inline docs/api/product_variants.rst]: **the** ProductVariant model

File: docs/api/products.rst [hunk 51]
```diff
@@ -8,57 +8,57 @@ Product API response structure
...
+| productTaxons | Collection of relations between product and taxons                         |
++---------------+----------------------------------------------------------------------------+
+| mainTaxon     | The main taxon to whose the product is assigned                            |
++---------------+----------------------------------------------------------------------------+
 
 
 .. note::
 
-    Read more about :doc: `Product model in the component docs</components/Product/models>`.
+    Read more about :doc:`Product model in the component docs</components/Product/models>`.
```
Comment (TheMadeleine) [inline docs/api/products.rst]: **the** Product model

File: docs/api/zones.rst [hunk 52]
```diff
@@ -0,0 +1,490 @@
...
+| name          | request        | Name of the zone                     |
++---------------+----------------+--------------------------------------+
+| scope         | request        | Scope of the zone                    |
++---------------+----------------+--------------------------------------+
+| members       | request        | Members of the zone                  |
++---------------+----------------+--------------------------------------+
+
+.. note::
+
+    Read more about :doc: `Zone types in the component docs</components/Addressing/zone_types>`.
```
Comment (TheMadeleine) [inline docs/api/zones.rst]: redundant space after `:doc:`

File: docs/api/tax_rates.rst [hunk 53]
```diff
@@ -0,0 +1,229 @@
...
++===============+================+======================================+
+| Authorization | header         | Token received during authentication |
++---------------+----------------+--------------------------------------+
+| code          | url attribute  | Code of the requested tax rate       |
++---------------+----------------+--------------------------------------+
+
+Example
+^^^^^^^
+
+To see the details of the the tax rate with ``code = clothing_sales_tax_7`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/tax_rates.rst]: `the the tax rate`

File: docs/api/taxons.rst [hunk 54]
```diff
@@ -348,37 +348,37 @@ Exemplary Response
...
 +---------------+----------------+--------------------------------------+
-| id            | url attribute  | Id of requested taxon                |
+| code          | url attribute  | Identifier of the requested taxon    |
 +---------------+----------------+--------------------------------------+
 
 Example
 ^^^^^^^
 
-To see the details for the the taxon with ``id = 9`` use the below method:
+To see the details of the the taxon with ``code = toys`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/taxons.rst]: `the the taxon` :rofl:

File: docs/api/zones.rst [hunk 55]
```diff
@@ -0,0 +1,490 @@
...
++===============+================+======================================+
+| Authorization | header         | Token received during authentication |
++---------------+----------------+--------------------------------------+
+| code          | url attribute  | Unique zone identifier               |
++---------------+----------------+--------------------------------------+
+
+Example
+^^^^^^^
+
+To see the details of the the zone with ``code = EU`` use the below method:
```
Comment (TheMadeleine) [inline docs/api/zones.rst]: `the the zone` :rofl:

File: docs/api/product_variants.rst [hunk 56]
```diff
@@ -522,165 +276,369 @@ Exemplary Response
...
++---------------+----------------+----------------------------------------------------------+
+| code          | url attribute  | Identifier of the product variant                        |
++---------------+----------------+----------------------------------------------------------+
+| productCode   | url attribute  | Id of product for which the variants should be displayed |
++---------------+----------------+----------------------------------------------------------+
+
+Example
+^^^^^^^
+
+To see the details of the the product variant with ``code = medium-theme-mug``, which is defined for the product with ``code = MUG-TH`` use the below method.
```
Comment (TheMadeleine) [inline docs/api/product_variants.rst]: `the the product variant` :rofl:

File: docs/api/payment_methods.rst [hunk 57]
```diff
@@ -0,0 +1,105 @@
+Payment Methods API
+===================
+
+These endpoints will allow you to easily manage payment methods. Base URI is `/api/v1/payment-methods`.
```
Comment (TheMadeleine) [inline docs/api/payment_methods.rst]: Shouldn't it be double backticks for all the URIs?

``` ``/api/v1/payment-methods`` ``` - looks better to me ;)

File: docs/api/payment_methods.rst [hunk 58]
```diff
@@ -0,0 +1,105 @@
...
+
+.. code-block:: bash
+
+     $ curl http://demo.sylius.org/api/v1/payment-methods/cash_on_delivery \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json"
+
+.. note::
+
+    The *cash_on_delivery* is just an example. Your value can be different.
```
Comment (TheMadeleine) [inline docs/api/payment_methods.rst]: The *cash_on_delivery* **code** is just an example.

Should be like that everewhere where you explain codes examples in those `.. note::` :)

**Comment by pjedrzejewski** [comment]
Thank you Ania!

**Pull request merged**